README.md

```diff
@@ -410,6 +410,47 @@ For example this means that by default we upgrade the minimum version of Airflow
 to 2.3.0 in the first Provider's release after 11th of October 2022 (11th of October 2021 is the date when the
 first `PATCHLEVEL` of 2.2 (2.2.0) has been released.
 
+Providers are often connected with some stakeholders that are vitally interested in maintaining backwards
+compatibilities in their integrations (for example cloud providers, or specific service providers). But,
+we are also bound with the [Apache Software Foundation release policy](https://www.apache.org/legal/release-policy.html)
+which describes who releases, and how to release the ASF software. The provider's governance model is something we name
+"mixed governance" - where we follow the release policies, while the burden of maintaining and testing
+the cherry-picked versions is on those who commit to perform the cherry-picks and make PRs to older
```

Comment:
minor suggestion - "while the responsibility of deciding cherry-picks for the older branches of a provider is on those who take ownership to maintain and test the cherry-picks against older versions of the given provider."

Reply:
> This is mostly dirty, boring and ground work to be done, nothing really "cool". I've been doing it for years, so I know what I am talking about :)

Fair enough :) Agree with your current language given the context you shared.